doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -172,6 +172,7 @@ Matter samples
 --------------
 
 * Added factory data, PICS and configuration overlay files to the :ref:`matter_weather_station_app` that are useful for getting to know the Matter certification process.
+* Updated ZAP configuration of the samples to conform with device types defined in Matter 1.0 specification.
```

Comment:
```suggestion
* Updated ZAP configuration of the samples to conform with device types defined in the Matter SDK v1.0 specification.
```

Reply:
Matter 1.0 specification is a separate thing from SDK. Also, the specification is officially advertised as "1.0", not "v1.0".